### PR TITLE
Bugfix/update ejs

### PR DESCRIPTION
**Pull request by lkappeler**
### Description

I tried to uptade ejs to version 2.5.7 to get rid of the security warnings mentioned in the connected issue.
Currently the tests are failing. The suggested linter does not give any hints.
Read the issue conversation for further information.

- connect to #280

### Checklist

- [ ] New tests added or existing tests modified to cover all changes
- [x] Code conforms with the [style
  guide](http://loopback.io/doc/en/contrib/style-guide.html)

### CODE REVIEW

**Comment by slnode** [comment]
Can one of the admins verify this patch? To accept patch and trigger a build add comment ".*ok\W+to\W+test.*"

**Comment by bajtos** [comment]
@slnode test please

**Comment by bajtos** [comment]
@lkappeler thank you for the pull request. First of all, please sign our Contributor License Agreement: https://cla.strongloop.com/agreements/strongloop/loopback-sdk-angular

File: lib/services.template.ejs [hunk 1]
```diff
@@ -7,13 +7,13 @@ if (typeof module !== 'undefined' && typeof exports !== 'undefined' &&
   //   import lbServices from './lb-services';
   //   angular.module('app', [lbServices]);
   //
-  module.exports = <%-: moduleName | q %>;
+  module.exports = <%-: helpers.quotedString(moduleName) %>;
```
Comment (bajtos) [inline lib/services.template.ejs]: I think you need to replace all occurences of `<%-:` with `<%-`. IIRC, the `:` suffix was enabling filters via the `|` operator. AFAICT, it's no longer available in the latest EJS version (see e.g. [here](https://www.npmjs.com/package/ejs#tags)).

Could you please try to make this change and see if it fixes the problem you are observing?

**Comment by lkappeler** [comment]
Thanks!, the code looks fine to me. I'm quiet busy right not, will test it with our application as soon as I'll find some time.

**Comment by bajtos** [comment]
Landed 🎉  Thank you for the contribution! ❤️

**Comment by bajtos** [comment]
Released in `loopback-sdk-angular@3.3.0`

**Comment by slnode** [comment]
Can one of the admins verify this patch? To accept patch and trigger a build add comment ".*ok\W+to\W+test.*"

**Pull request merged**